### DISCUSSION TITLE
What is the timeline for the next PyPI release?

### DISCUSSION THREAD

**Discussion by eindenbom** [openai/whisper#1803] [Q&A]
What is the timeline for the next PyPI release?

**Comment by jongwook** [comment]
Thanks for your PR - I'm planning one this week but you can use:

    git+https://github.com/openai/whisper.git@1cea4357687b676b293cb5473e1ade25f5b1cef7

in the meantime.

**Comment by eindenbom** [comment]
Thank you. I'll wait for PyPI package.
